css-basics/css-grid/css-grid_place-elements/css/styles.css

```diff
@@ -0,0 +1,106 @@
+/*
+Use grid to fulfill the following tasks:
+
+[ ] - .element__1 should start at column three and go until column five
+[ ] - .element__2 should start at column five and go until the end
+[ ] - .element__5 should span two columns and two rows
+[ ] - .element__8 should span two rows
+[ ] - .element__spiced should be in row one and span the entire grid width
+[ ] - .element__20 should start at row six and span two rows
+
+*/
+:root {
+  --primary-dark: #2e016d;
+  --primary: #3f0097;
+  --primary-light: #8236ff;
+
+  --secondary-dark: #38ccda;
+  --secondary: #7fd9e2;
+
+  --accent: #ffd15d;
+  --black-light: #303030;
+  --black: #000000;
+  --white: #ffffff;
+  --grey-dark: #b6b9c2;
+  --grey-light: #f5f6f6;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: system-ui;
+  background-color: var(--grey-light);
+}
+
+.container {
+  display: grid;
+  gap: 20px;
+  grid-template-columns: repeat(5, 1fr 2fr);
+  grid-template-rows: repeat(10, 50px);
+  padding: 10px;
+}
+
+.container__element {
+  padding: 10px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: var(--primary-light);
+  border-radius: 3px;
+  font-size: 2rem;
+}
+
+.element-1,
+.element-2,
+.element-5,
+.element-8,
+.element-20,
+.element-spiced {
+  background-color: var(--primary-dark);
+  color: var(--white);
+}
+
+.element-1 {
+  /* Add styles here */
+  grid-column-start: 3;
+  grid-column-end: 5;
+}
+
+.element-2 {
+  /* Add styles here */
+  grid-column: 5 / -1;
+}
+
+.element-5 {
+  /* Add styles here */
+  grid-column-start: 3;
+  grid-column-end: 5;
+  grid-row-start: 3;
+  grid-row-end: 5;
+}
+
+.element-8 {
+  /* Add styles here */
+  grid-column-start: 6;
+  grid-column-end: 7;
+  grid-row-start: 3;
+  grid-row-end: 5;
+}
+
+.element-spiced {
+  /* Add styles here */
+  grid-column: 1 / -1;
+  grid-row: 1;
+}
+
+.element-20 {
+  /* Add styles here */
+  grid-row-start: 7;
```

Comment:
Glaube, das stimmt nicht. Das Element sollte in der 6. und nicht 7. Reihe starten. Und auch hier geht Span wieder. :)